trinity/main.py

```diff
@@ -250,3 +274,11 @@ def run_service_until_quit(service: BaseService) -> None:
     asyncio.ensure_future(service.run())
     loop.run_forever()
     loop.close()
+
+
+def setup_plugins() -> PluginManager:
+    plugin_manager = PluginManager()
+    # TODO: Implement auto-discovery of plugins based on some convention/configuration scheme
```

Comment:
I think this is what we want to do.

https://packaging.python.org/guides/creating-and-discovering-plugins/

Specifically this section

https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata

Reply:
Thanks for the pointer! Will look into it.